<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="32840" started-at="2019-11-12T17:17:55 IST" finished-at="2019-11-12T17:18:27 IST">
    <groups>
    </groups>
    <test name="Default test" duration-ms="32840" started-at="2019-11-12T17:17:55 IST" finished-at="2019-11-12T17:18:27 IST">
      <class name="com.training.sanity.tests.ELTC_016">
        <test-method status="PASS" signature="setUpBeforeClass()[pri:0, instance:com.training.sanity.tests.ELTC_016@6321e813]" name="setUpBeforeClass" is-config="true" duration-ms="19" started-at="2019-11-12T17:17:55 IST" finished-at="2019-11-12T17:17:55 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpBeforeClass -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.training.sanity.tests.ELTC_016@6321e813]" name="setUp" is-config="true" duration-ms="15215" started-at="2019-11-12T17:17:55 IST" finished-at="2019-11-12T17:18:10 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="Logout()[pri:0, instance:com.training.sanity.tests.ELTC_016@6321e813]" name="Logout" duration-ms="9" started-at="2019-11-12T17:18:10 IST" finished-at="2019-11-12T17:18:10 IST">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.training.generics.GenericMethods.getElement(GenericMethods.java:34)
at com.training.sanity.tests.ELTC_016.Logout(ELTC_016.java:68)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Logout -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.training.sanity.tests.ELTC_016@6321e813]" name="setUp" is-config="true" duration-ms="13453" started-at="2019-11-12T17:18:10 IST" finished-at="2019-11-12T17:18:23 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="validLoginTest()[pri:0, instance:com.training.sanity.tests.ELTC_016@6321e813]" name="validLoginTest" duration-ms="4057" started-at="2019-11-12T17:18:23 IST" finished-at="2019-11-12T17:18:27 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validLoginTest -->
      </class> <!-- com.training.sanity.tests.ELTC_016 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
